File: scanner/corpora/csharp/positive/hmacsha256.cs
using System;
using System.Security.Cryptography;
using System.Text;

class HmacSha256Example {
    static void Main() {
        byte[] key = Encoding.UTF8.GetBytes("my-secret-key");
        byte[] message = Encoding.UTF8.GetBytes("important data");

        using (HMACSHA256 hmac = new HMACSHA256(key)) {
            byte[] hash = hmac.ComputeHash(message);
            Console.WriteLine(BitConverter.ToString(hash).Replace("-", ""));
        }
    }
}

File: scanner/corpora/python/positive/hmac_sha256.py
"""HMAC-SHA256 message authentication using Python hashlib."""
import hmac
import hashlib

key = b"my-secret-key"
message = b"important data to authenticate"
digest = hmac.new(key, message, hashlib.sha256).hexdigest()
print(f"HMAC-SHA256: {digest}")
